declarations/LoaderContext.d.ts

```diff
@@ -0,0 +1,172 @@
+import type { AssetInfo, Configuration } from "../lib";
+import Compilation from "../lib/Compilation";
+import NormalModule, { InputFileSystem } from "../lib/NormalModule";
+import type { Mode } from "./WebpackOptions";
+
+export interface LoaderContext {
+	version: number;
+	getOptions(schema: any): any;
+	emitWarning(warning: Error | string): void;
+	emitError(error: Error | string): void;
```

Comment:
```suggestion
	emitWarning(warning: Error): void;
	emitError(error: Error): void;
```

Reply:
Looking at usage it seems these can be `strings`:

https://github.com/webpack/webpack/blob/85fe6ac4f436384e2a20c7996c70f14e4172ccd8/lib/NormalModule.js#L521

```js
			emitError: error => {
				if (!(error instanceof Error)) {
					error = new NonErrorEmittedError(error);
				}
				this.addError(
					new ModuleError(error, {
						from: getCurrentLoaderName()
					})
				);
			},
```